### PR TITLE
MID track reader spec

### PR DESCRIPTION
**Pull request by dstocco**

### CODE REVIEW

**Comment by dstocco** [comment]
Hi @shahor02 ,
I've just added the TrackReaderSpec. Please let me know if I have to add it to the `inputHelper` as well.

File: Detectors/MUON/MID/Workflow/src/tracks-reader-workflow.cxx [hunk 1]
```diff
@@ -13,114 +13,22 @@
...
-
-  /// publish the data and clear the internal vector
-  void publish(DataAllocator& out)
-  {
-    out.snapshot(OutputRef{"rofs"}, mROFs);
-    out.snapshot(OutputRef{"tracks"}, mTracks);
-    mROFs.clear();
-    mTracks.clear();
-  }
+  workflowOptions.emplace_back("disable-mc", VariantType::Bool, true, ConfigParamSpec::HelpString{"Disable MC info"});
```
Comment (shahor02) [inline Detectors/MUON/MID/Workflow/src/tracks-reader-workflow.cxx]: the standard policy is to have `"disable-mc", VariantType::Bool, false`. Then, in the raw data processing it is set to true via option from env.var common for all workflows.

**Review by shahor02 — COMMENTED** [review]
thanks, will check in details bit later, at the moment 1 comment

**Comment by shahor02** [comment]
Thanks! Since meanwhile I had some fixes in my commits, to avoid conflict resolution/merging I 've cherr-picked your commits, please don't be scared by closing the PR :)

**Pull request closed without merging**